base/build-static-ioc.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+set -ex
+
+IOC=$1
```

Comment:
`$1` may be undefined, and the shell will happily accept it until `tar` fails to find the directory. I know this argument is clearly documented in the README, but it should be clear to understand what's going on when it is forgotten.

Adding `nounset (-u)` option above could fix this.

Reply:
I thought I had added `-u`, actually. Thanks for spotting it :)